README.md

```diff
@@ -1,10 +1,10 @@
 # Making Layer Cakes
 
-Use generators for class/traits-like composition of objects-as-closures
+Helpers for class/traits-like composition of objects-as-closures
 
 ## The Objects-as-closures Pattern
 
-The classic object or object-oriented programming has public methods and private instance variables. There are several ways to encode this in JavaScript. The following code uses JavaScript's class syntax with the new support for private instance variables.
+The classic object or object-oriented programming has public methods and private instance variables. There are several ways to encode this in JavaScript. The following code uses JavaScript's class syntax with the [new support for private instance variables](https://github.com/tc39/proposal-private-methods).
```

Comment:
That link is specifically private methods, rather than private state variables. Use https://github.com/tc39/proposal-class-fields#private-fields instead.